ui/packages/catalog-ui-search/src/main/webapp/js/widgets/openlayers.polygon.js

```diff
@@ -50,19 +50,49 @@ const translateFromOpenlayersCoordinates = coords => {
     .flatten()
 }
 
+const isValidBufferWidth = bufferWidth =>
+  !_.isNull(bufferWidth) && !_.isUndefined(bufferWidth) && '' != bufferWidth
+
 /**
- * Returns buffer width in meters with the lowest value of 1 meter.
+ * Returns buffer width in meters, if applicable, with the lowest value of 1 meter.
  */
 const getConstrainedBufferWidth = model => {
+  const polygonWidthValue = model.get('polygonBufferWidth')
+
+  if (!isValidBufferWidth(polygonWidthValue)) return undefined
+
+  if (0 == polygonWidthValue) return undefined
```

Comment:
✏️ This seems readable to me, but I know some folks don't like single-line `if`s without brackets.

Reply:
If this doesn't bother you guys then I won't make a fuss, but I personally don't like debugging through code and losing my current context without clearly seeing what the result of the `if` statement was. Sure you could infer it from your loss of context but that strikes me as error prone.

Reply:
Sorry, @Lambeaux are you talking about the lack of `{}` here? I'm not quite following.

Reply:
Yeah I'm talking directly to the behavior of (at least the Java) debugger when returns are on the same line as control flow decisions.